lib/LMSManagers/LMSHelpdeskManager.php

```diff
@@ -1086,9 +1086,9 @@ public function TicketAdd($ticket, $files = null)
 			netnodeid, netdevid, verifierid, deadline, service, type, invprojectid, parentid)
 				VALUES (?, ?, ?, ?, ?, ?, ?, 0, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)', array($ticket['queue'],
             empty($ticket['customerid']) ? null : $ticket['customerid'],
-            $ticket['requestor'],
-            $ticket['requestor_mail'],
-            $ticket['requestor_phone'],
+            isset($ticket['requestor']) ? $ticket['requestor'] : null,
```

Comment:
A co powiesz na to:
https://github.com/chilek/lms-plus/blob/c46412fac6f1dbfdda831b6e63c64aef00bca1e9/doc/lms.pgsql#L1922
;-) ?

Reply:
Od razu to rozwikłam może - co daje `DEFAULT ' NOT NULL'` zamiast samego `DEFAULT NULL`?
Czy to są jakieś zaszłości zaszłości czy może ma to jakieś uzasadnienie?

Reply:
łatka na db: https://github.com/lmsgit/lms/pull/2001/commits/0f5ae2e9db6a8bd2301bf3c71a2073c8d80658a7